docs/core/run-time-config/globalization.md

```diff
@@ -76,3 +76,14 @@ Project file:
 | - | - | - |
 | **runtimeconfig.json** | `Switch.System.Globalization.FormatJapaneseFirstYearAsANumber` | `false` - format as "Gannen"<br/>`true` - format as number |
 | **Environment variable** | N/A | N/A |
+
+## NLS
+
+- Determines whether .NET uses National Language Support (NLS) or International Components for Unicode (ICU) globalization APIs for Windows apps. .NET 5.0 and later versions use ICU globalization APIs by default on Windows 10 May 2019 Update and later versions.
+- Default: Use ICU globalization APIs (`false`).
```

Comment:
This should instead be: 
```suggestion
- Default: Use NLS globalization APIs (`false`).
```

or 

```suggestion
- Default: Use ICU globalization APIs (`true`).
```

Reply:
This is trying to show the default value if you don't explicitly set it, which is to use ICU, correct?

Reply:
Right, if the switch is not present, the default behavior is to use ICU.

Reply:
So should I leave the default out altogether? Your suggestion is not making sense to me.

Reply:
Now I understand what this is saying. However the (`false`) inside the parenthesis was confusing.

but this is saying that the default is to use ICU... maybe remove the `false` part?

Reply:
Thanks @safern. I agree that it might be confusing as written. Just to understand better, what did you think the `false` referred to? I'm trying to devise a better way to display defaults for this setting and all the others that currently list it in the same way.

Reply:
I understood `false` referred to `Use ICU globalization APIs` — but after your response I was then able to understand it referred to using NLS and that the default was referring to it using ICU. 

Does that make sense?

Reply:
Yes, that makes sense, thanks!